utils/service_discovery/abstract_config_store.py

```diff
@@ -61,6 +65,28 @@ def client_read(self, path, **kwargs):
     def dump_directory(self, path, **kwargs):
         raise NotImplementedError()
 
+    def _populate_image_to_checks(self):
+        """Populate the image_to_checks cache with templates pulled
+        from the config store and from the auto-config folder"""
+        images_to_checks = defaultdict(set)
```

Comment:
Nitpick: we could call the variable `image_to_checks` to be consistent with `self.image_to_checks` :)